packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -7,29 +7,1024 @@ category: management
 
 <ApiLifecycle access="ea" />
 
+> **Note:** The MyAccount API is being enhanced, accessible at `/api/idp/myaccount`. Use this endpoint if you plan to use the MyAccount API. The existing endpoint (`/api/v1/myaccount`) is deprecated.
+>
+>Also, to use the new endpoint, move the `version` from the path to the `Accept` header. Use a value of `application/json; okta-version=1.0.0`.
+
 The Okta MyAccount API allows end users (with or without administrator access) to fetch and update their own Okta user profiles.  It implements a subset of the existing [Users API](/docs/reference/api/users/) but with significant differences.  This API does not expose information an end user should not have access to, and it does not support lifecycle operations.
```

Comment:
same here, we should remove '(with or without administrator access)'. for update, myaccount api only supports non-admin.
```suggestion
The Okta MyAccount API allows end users to fetch and update their own Okta user profiles.  It implements a subset of the existing [Users API](/docs/reference/api/users/) but with significant differences.  This API does not expose information an end user should not have access to, and it does not support lifecycle operations.
```